core/state/intra_block_state_test.go

```diff
@@ -535,122 +534,3 @@ func TestIntraBlockStateNewContractAccount(t *testing.T) {
 	}
 
 }
-
-// TestCopy tests that copying a statedb object indeed makes the original and
-// the copy independent of each other. This test is a regression test against
-// https://github.com/ethereum/go-ethereum/pull/15549.
-func TestCopy(t *testing.T) {
```

Comment:
Removed this test because it is the only place that uses tries concurrently, and therefore requires locking in `trie_pruning.go`, at a great performance cost